test_app/tests.py

```diff
@@ -538,13 +538,13 @@ def _parse_to(to):
                 if not name:
                     ret.append(address)
                 else:
-                    ret.append("%s <%s>" % (name, address))
+                    ret.append("\"%s\" <%s>" % (name, address))
```

Comment:
Yep. I thought the similar things, too: doing it in two places just means they match. One to revisit, post-bugfix